webapp/package.json

```diff
@@ -17,6 +17,7 @@
     },
     "dependencies": {
         "@primer/octicons-react": "10.1.0",
+        "@testing-library/react": "12.1.2",
```

Comment:
This is an awesome and welcomed change to the project. I'm thinking the significance of the change should be more prominently known. We can repurpose the PR as "Add `@testing-library/react` and component tests", and treat the typescript migration pieces as a requirement in the same PR that is used to show proof that the testing library is set up properly. What do you think?

Reply:
We can add this to `devDependencies`, and also get rid of the dependencies related to `enzyme`